src/Components/DashboardMenu/DashboardMenu.tsx

```diff
@@ -1,14 +1,18 @@
-import React from "react";
+import React, { useState } from "react";
 import { NavLink, useLocation } from "react-router-dom";
 import './Dashboard.css'
 
 const DashboardMenu = () => {
+  const [active, setActive] = useState('1')
+  const handleClick = (event: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
+    setActive(event.currentTarget.id)
+  }
   return(
     <div className="user-dash-nav">
-      <NavLink to='/dashboard'><button className="nav-button-mybooks">My Books</button></NavLink>
-      <NavLink to='/dashboard/my-borrowed-books'><button className="nav-button-mybooks">My borrowed books</button></NavLink>
-      <NavLink to='/dashboard/pending-requests'><button className="nav-button-mybooks">Pending Requests</button></NavLink>
-      <NavLink to='/dashboard/add-book'><button className="nav-button-mybooks">Add a book</button></NavLink>
+      <NavLink to='/dashboard'><button id='1' className={active === '1' ? "nav-button-mybooks active" : "nav-button-mybooks"} onClick={(event) => handleClick(event)}>My Books</button></NavLink>
```

Comment:
Good way of determining if the button is active or not! I think that navLinks have inherent abilities to be active or not, butI'm not sure how to style that. I think using NavLink's features is probably a good extension!